externs/shaka/offline.js

```diff
@@ -123,6 +123,7 @@ shaka.extern.ManifestDB;
  *   hdr: (string|undefined),
  *   kind: (string|undefined),
  *   language: string,
+ *   originalLanguage: ?string,
```

Comment:
I'm pretty sure this should be `(string|undefined)`, instead of `?string`.
`?string` suggests that this is a string or null. However, in the case of an offline asset that was stored before `originalLanguage` was a field, `originalLanguage` would be `undefined`. So unless we are planning on migrating over to a version 6 of the storage cell system, any new fields should have `undefined` as an option.

Reply:
Sorry, wasn't so aware of offline requirements. Fix is live.